source/includes/cli-reference/_projectfile.html.md

```diff
@@ -0,0 +1,63 @@
+# Projects and Config Files
+
+The Bonsai command-line interface works with three files.  A `.bonsai` file in
+your user directory, a project file in the directory containing the Inkling 
+files and simulator configuration for a BRAIN, and a `.brains` file that links
+a project file to a BRAIN in the same directory as the project file.
+
+## `.bonsai`

```

Comment:
I'm trying to think of a good way to make these three sub headings have some kind of parity, also I think code formatted text in a heading looks odd. These file names aren't things you type into a command prompt.

What about:
# Projects and Config Files
## .bproj file
## .bonsai file
## .brains file

(no ``)